src/Command/ModifyEntity/LineItem/LineItemChangeStateCommand.php

```diff
@@ -0,0 +1,226 @@
+        $this->setHelp(
+            <<<EOF
+The <info>%command.name%</info> command Activate/Deactivate line items into the application.
+
+    <info>php %command.full_name% <path></info>
+
+To Activate a line item by slug:
+
+    <info>php %command.full_name% activate slug {line-item-slug}</info>
+    
+To Activate a line item by multiple slugs:
+
+    <info>php %command.full_name% activate slug {line-item-slug1} {line-item-slug2}</info>
+
+To Activate a line item by id:
+
+    <info>php %command.full_name% activate id {line-item-id}</info>
+
+To Activate a line item by uri:
+
+    <info>php %command.full_name% activate uri {line-item-id}</info>
+To Deactivate a line item by slug:
+
+    <info>php %command.full_name% deactivate slug {line-item-slug}</info>
+    
+To Deactivate a line item by id:
+
+    <info>php %command.full_name% deactivate id {line-item-id}</info>
+
+To Deactivate a line item by uri:
+
+    <info>php %command.full_name% deactivate uri {line-item-id}</info>
+EOF
+        );
+
+        $this->addArgument(
+            'toggle',
+            InputArgument::REQUIRED,
+            'Accepted two values "activate" to activate a line item. "deactivate" to deactivate a line item.'
+        );
+
+        $fieldQueryDescription = 'How do you want to query the line items that you want to activate/deactivate. '
+            . 'Accepted parameters are: %s';
+        $this->addArgument(
+            'query-field',
+            InputArgument::REQUIRED,
+            sprintf($fieldQueryDescription, implode(', ', self::AVAILABLE_QUERY_FIELDS))
+        );
+
+        $fieldValueDescription = 'The value that should match based on the query field. it can be one value or a list'
+            . ' of values split by space. Example: given that the query'
+            . ' field is "slug" and the query value is "test1 test2" then all the line items '
+            . 'with slug equals to test1 or test2 will be updated';
+        $this->addArgument(
+            'query-value',
+            InputArgument::REQUIRED | InputArgument::IS_ARRAY,
+            $fieldValueDescription
+        );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        parent::initialize($input, $output);
+
+        $this->symfonyStyle = new SymfonyStyle($input, $output);
+
+        $this->symfonyStyle->title('Simple Roster - Line Item Activator');
+    }
+
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $toggle = (string)$input->getArgument('toggle');
+        $queryField = (string)$input->getArgument('query-field');
+        $queryValue = $input->getArgument('query-value');
+
+        $this->validateToggleArgument($toggle);
+        $this->validateQueryFieldArgument($queryField);
+
+        try {
+            $this->symfonyStyle->comment(sprintf('Executing %s...', ucfirst($toggle)));
```

Comment:
I do not believe so because execute doesn't mean that you're starting something it just means that something is being executed. Also, this comment depends on the informed argument which requires to be validated first. So, if you don't mind, I'd like to keep it like this.

Reply:
Agree. But for me it looks like:
you're _already_ executing command (parameter validation is a part of execution), and actions called, can already throw exceptions. And after actual (not before, which, IMO - makes more sense) start you're pushing a message that you're executing smth.
It's just a matter of taste, but I'd do it next:
a) at the very start of `execute` method, I'd pushed a message, that command is now being executed;
b) re-phrased this message, leaving it at its place, with smth like "Starting lineItem(-s) %s"

Reply:
p.s.
I'm not strictly pushing for this exact change, I just want to see you concerns, after a 2nd explanation, plus, @luisfn  opinion :) . So, if you will either convince me, or Luis will also think that it can be as is - we can keep it ;)

Reply:
For me it seems fine as is